vcd/resource_vcd_vapp_multi_vm_in_template_test.go

```diff
@@ -0,0 +1,186 @@
+		"VmNameInTemplate":   testConfig.VCD.Catalog.VmName1InMultiVmItem,
+		"VmNameInTemplate2":  testConfig.VCD.Catalog.VmName2InMultiVmItem,
+		"VappName":           vappName,
+		"VmName":             vmName,
+		"VmName2":            vmName2,
+		"ComputerName":       vmName + "-unique",
+		"Tags":               "vapp vm",
+		"OvaPath":            testConfig.Ova.OvaVappMultiVmsPath,
+	}
+
+	var configText string
+	if testConfig.VCD.Catalog.CatalogItemWithMultiVms == "" {
+		configText = templateFill(defaultCatalogItem+testAccCheckVcdVAppVmMultiVmInTemplate, params)
+	} else {
+		configText = templateFill(testAccCheckVcdVAppVmMultiVmInTemplate, params)
+	}
+
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	debugPrintf("#[DEBUG] CONFIGURATION: %s\n", configText)
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckVcdVAppVmDestroy(vappName),
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: configText,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckVcdVAppVmExists(vappName, vmName, "vcd_vapp_vm."+vmName, &vapp, &vm),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName, "name", vmName),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName, "computer_name", vmName+"-unique"),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName, "network.0.ip", "10.10.102.161"),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName, "power_on", "true"),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName, "metadata.vm_metadata", "VM Metadata."),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName2, "name", vmName2),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName2, "computer_name", vmName+"-unique"),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName2, "network.0.ip", "10.10.102.162"),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName2, "power_on", "true"),
+					resource.TestCheckResourceAttr(
+						"vcd_vapp_vm."+vmName2, "metadata.vm_metadata", "VM Metadata."),
+				),
+			},
+		},
+	})
+}
+
+const defaultCatalogItem = `
+resource "vcd_catalog_item" "defaultOva" {
+  org     = "{{.Org}}"
+  catalog = "{{.Catalog}}"
+
+  name                 = "TestAccVcdVAppMultiVmInTemplate"
+  ova_path             = "{{.OvaPath}}"
+}
+`
+
+const testAccCheckVcdVAppVmMultiVmInTemplate = `
+resource "vcd_network_routed" "{{.NetworkName}}" {
+  name         = "{{.NetworkName}}"
+  org          = "{{.Org}}"
+  vdc          = "{{.Vdc}}"
+  edge_gateway = "{{.EdgeGateway}}"
+  gateway      = "10.10.102.1"
+
+  static_ip_pool {
+    start_address = "10.10.102.2"
+    end_address   = "10.10.102.254"
+  }
+}
+
+resource "vcd_vapp" "{{.VappName}}" {
+  name = "{{.VappName}}"
+  org  = "{{.Org}}"
+  vdc  = "{{.Vdc}}"
+}
+
+resource "vcd_vapp_org_network" "vappNetwork1" {
```

Comment:
HCL scripts in this file are not formatted.

Reply:
fixed tab issue